### ISSUE TITLE
Invalid JSON for artifact

### ISSUE THREAD

**Issue by IgorBalog-Eng** [International-Data-Spaces-Association/IDS-testbed#88]
Example payload which is used in testbed, is not valid JSON. 
It contains invalid characters, like ‘ and ’, and when you decode payload from DSC, and try to format JSON, it will not be possible.

```
"{‘type’:’FeatureCollection’,’features’:[{‘type’:’Feature’,’id’:’Autowarn_Analyse.fid-46a27240_180cc7856b1_-1fb6’,
```

When executing preconfigure request
Artifacts: POST /api/artifacts
this one will insert invalid json string to DSC provider connector.

Possible solution:
update postman collection, and replace existing invalid json string with correct one.

This one does not impact on Testbed functionality, since no real payload processing is done.

**Comment by jfernandezsqs** [comment]
It should be solved in the PR #78

**Comment by jfernandezsqs** [comment]
Thanks for reporting the issue Igor!
PR #78 merged. This issue is solved, therefore I close it.
